internal/cli/cmd/clustermesh.go

```diff
@@ -162,6 +162,7 @@ func newCmdClusterMeshStatus() *cobra.Command {
 	cmd.Flags().BoolVar(&params.Wait, "wait", false, "Wait until status is successful")
 	cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 15*time.Minute, "Maximum time to wait")
 	cmd.Flags().BoolVar(&params.SkipServiceCheck, "skip-service-check", false, "Do not require service IP of remote cluster to be available")
+	cmd.Flags().StringVarP(&params.Output, "output", "o", "compact", "Output format. One of: (json, compact).")
```

Comment:
That was just to give a name to the default. Should I name it `summary` instead, or just use an empty string?

Reply:
I meant that the default option is compact/summary so users needn't specify it by passing `-o compact`. Users will use `-o` only when they want to print the output in the json format.

Reply:
OK, so then I'd check against the empty string for the default value?

Reply:
IMO, the default value is automatically added in help command by cobra, which makes it clear enough to user to know all applicable values, and they can also omit specifying default value (e.g. summary).

```
  -o, --output string            Output format. One of: json, summary (default). (default "summary")
```

Reply:
OK. Sorry, it took me a while to understand. I'm guessing what you and @aditighag mean is:

```go
cmd.Flags().StringVarP(&params.Output, "output", "o", "summary", "Output format. One of: json.")
```

since `summary` will be mentioned anyway, right?